pkg/apis/app/v1alpha1/kogitoservices_types.go

```diff
@@ -193,6 +194,13 @@ type KogitoServiceSpec struct {
 	// Create Service monitor instance to connect with Monitoring service
 	// +optional
 	Monitoring Monitoring `json:"monitoring,omitempty"`
+
+	// +optional
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.displayName="Configs"
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:com.tectonic.ui:text"
+	// Application properties that will be set to the service. For example 'MY_VAR: my_value'.
```

Comment:
Shouldn't be the example rather `my_var=my_value`?

Reply:
The properties are set directly to the yaml file like:

```yaml
config:
    my.application.key: value
```

So it makes sense.

Reply:
right, I see